core/storage/question/gae_models.py

```diff
@@ -54,6 +54,9 @@ class QuestionModel(base_models.VersionedModel):
         required=True, indexed=True)
     # The ISO 639-1 code for the language this question is written in.
     language_code = ndb.StringProperty(required=True, indexed=True)
+    # The skill ids linked to this question.
+    linked_skill_ids = ndb.StringProperty(
+        required=False, indexed=True, repeated=True)
```

Comment:
You can drop the "required" field altogether for repeated properties. (Sorry, I had forgotten about that.) See https://cloud.google.com/appengine/docs/standard/python/ndb/entity-property-reference#options -- False is already the default.

Reply:
Done